### ISSUE TITLE
SciQA image Evaluation on InstructBLIP (Vicuna-7B) Help

### ISSUE THREAD

**Issue by weiyueli7** [salesforce/LAVIS#390]
Hello, 

If I understand the process from the paper correctly, we are using the 2017 images with their corresponding questions from the test set of [Science QA](https://drive.google.com/drive/folders/1w8imCXWYn2LxajmGeGH_g5DaL2rabHev) to evaluate the InstructBLIP (Vicuna-7B) model. However, after trying out for a day, I can only achieve **48.04** using the pre-trained weights as opposed to what the paper claimed as **60.5**. Could anyone help me see if my prompt format is off?

I use the below format to feed into the model (an example of 2877 in the test set of Science QA):

```
Question: {Think about the magnetic force between the magnets in each pair. Which of the following statements is true?} Options: {(a) The magnetic force is weaker in Pair 2. (b) The strength of the magnetic force is the same in both pairs. (c) The magnetic force is weaker in Pair 1.}. Answer:
```

I think I have closely followed the Instructions for Zero-shot Inference in the appendix, so any help is greatly appreciated!

**Comment by chenllliang** [comment]
Did you try adding the vocab constraints and ranking?

**Comment by DoggyLu** [comment]
你好，你能简单的说一下怎么复现的instructblip的呢，是需要编写lavis的yaml文件吗？如果是的话，可以给个示例吗

**Comment by Zhudongsheng75** [comment]
I have the same question. Maybe 'Context' is the import key in scienceqa task, but I don't know how to construct it.

**Comment by mikeshihyaolin** [comment]
Could the authors provide evaluation script or  implementation guideline for ScieceQA?
